io.vikunja.Desktop.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.vikunja.Desktop
+app-version: '0.21.0'
+version-hash: '46bd28e13d35f78bbcf2d3b44307059b195f5d1eefcd08937645888f008365e3'
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+command: vikunja-desktop
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
does it really support wayland out of the box?

Reply:
Also `--share=ipc` is necessary.
and I suspect `--device=dri`

Reply:
It's an electron app, AFAIK it does.

Reply:
afaik electron usually doesn't.

Reply:
Since there is an x11 fallback (which I added), it's fine

Reply:
no it's not. 

the fallback is for the case when the host doesn't have wayland.

Reply:
removed